manager_f1tenth.py

```diff
@@ -13,19 +29,19 @@ def env_setup():
     os.system(
         "git clone --quiet https://github.com/f1tenth/f1tenth_labs_openrepo.git > /dev/null"
```

Comment:
Also, I was thinking that we can make the URL to f1tenth_labs_openrepo link configurable, so that one can link their own fork of the repo here and can directly push their code to their own github repository.

Reply:
That's a great idea. Let's make it a separate issue/pr.